core/src/data/value/mod.rs

```diff
@@ -432,6 +432,7 @@ mod tests {
     use {
         super::{Interval, Value::*},
         crate::data::value::uuid::parse_uuid,
+        crate::data::ValueError,
```

Comment:
This could be merge by
```rust
crate::data::{
    value::uuid::parse_uuid,
    ValueError,
};
```

This is a very nit comment and also it is in the unit tests, so I will approve and merge this.
You can consider this for the next time.